src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const data = [...document.querySelectorAll('.population')];
```

Comment:
Ensure that elements with the class 'population' exist in the DOM to avoid potential errors when mapping over an empty array.